docs/docs/terra.md

```diff
@@ -0,0 +1,296 @@
+# WorkFlow Launcher's Role in Terra
+
+## Summary
+    
+The Data Sciences Platform (DSP) is building a new system (around
+[Terra](https://terra.bio/)) for storing and processing biological
+data. The system design includes a Data Repository where data is
+stored, and a Methods Repository that executably describes
+transformations on that data.
+
+In the new system, the DSP needs something to fulfill the role that
+Zamboni currently plays in DSP's current infrastructure to support
+the Genomics Platform (GP).
+
+Zamboni watches various queues for messages describing new data and
+how to process it. Zamboni interprets those messages to dispatch
+work to workflow engines (running on the premises or in the cloud)
+and monitors the progress of those workflows. The Zamboni web UI
+allows users to track the progress of workflows, and enables Ops
+engineers to debug problems and resume or restart failed workflows.
+Zamboni can run workflows on both a local Sun Grid Engine (SGE), and
+on Cromwell on premises and in the cloud.
+
+We think that WFL can fill the role of Zamboni in the new data
+storage and processing system that DSP is developing now.
+
+## History
+    
+WFL began as a project to replace a Zamboni *starter*, with the old
```

Comment:
Same comment as before - do future users need to know about Zamboni? If so, then keep this. If not, then it will just confuse them.